README.md

```diff
@@ -120,10 +120,11 @@ Pymilo is an open source Python package that provides a simple, efficient, and s
 | scikit-learn | PyTorch | 
 | ---------------- | ---------------- | 
 | Linear Models &#x2705; | - | 
-| Neural networks &#x274C; | -  | 
+| Neural networks &#x2705; | -  | 
 | Clustering &#x274C; | -  | 
 | Trees &#x274C; | -  | 
 | Ensemble Models &#x274C; | - | 
+Details are available in `supported-models.md` file.
```

Comment:
Details are available in [Supported Models](https://github.com/openscilab/pymilo/blob/main/SUPPORTED_MODELS.md).

Reply:
My bad, sure.